README.md

```diff
@@ -1,24 +1,181 @@
 
-* Deployment instructions
+**You can do the following with app:**
 
-* ...
+- Create an account
+- Login
+- Create a category
+- Create a transaction
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+  * An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+  * [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+  * Make sure you have Ruby installed in your system. You can install it [here](https://www.ruby-lang.org/en/documentation/installation/)
+
+  * Get started with [Ruby on Rails](https://guides.rubyonrails.org/getting_started.html).
+
+  * Make sure you have [PostgreSQL](https://www.postgresql.org/) installed and running.
+
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```
+git clone https://github.com/ndikumanaisaie/Budget_App.git
+```
+
+  * Navigate to the location of the folder in your machine:
+
+  **`you@your-Pc-name:~$ cd Bugdet_App`**
+
+### Install
+
+To install all gems, run:
+
+```
+bundle install
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+  * After Cloning this repo to your local machine.
+  * To get it running in your terminal, run it in your IDE
+
+### Live version
+
+  * [Personal Finances](https://ndikumana.onrender.com/)
+### Demo Video
+
+  * [Personal Finances](https://www.loom.com/share/5d823c925f664d96850ee77d5dbd8449)
+
+### Running the app
+
+  *  Run rails `db:create:all` and `rails db:migrate`
+  *  Run `rails db:seed` to populate the database with some sample data.
+  *  Run `rails s` to start the server.
+  *  In your browser, go to `http://localhost:3000`.
+
+### Run tests
+
+  * rspec
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤 Authors <a name="authors"></a>
+
+👤 **Ndikumana Isaie**
+
+- GitHub: [ndikumanaisaie](https://github.com/ndikumanaisaie)
+- Twitter: [Ndikuma38670724](https://twitter.com/Ndikuma38670724)
+- LinkedIn: [Ndikumana Isaie](https://www.linkedin.com/in/ndikumanaisaie/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement more methods.**
+- [ ] **Write Extra Message.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project and how I managed to build it!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Big up to Gregoire Vella who created thiss nice looking design.
```

Comment:
[OPTIONAL] You can improve your acknowledgment by adding links to the author and design so that users can be able to view the design source, author's page, and the creative commons license as well.

This is how you can improve your acknowledgment:

- I would like to thank [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) from which I took inspiration for this beautiful design in accordance with the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).